### DISCUSSION TITLE
Does option --to-v1.1 overwrite ID3v1 tags if ID3v2 are empty?

### DISCUSSION THREAD

**Discussion by megalithicportal** [nicfit/eyeD3#578] [Q&A] [answered]
Hello, 
Does the eyeD3 command line option  --to-v1.1 overwrite ID3v1 tags even if the ID3v2 tags are blank/empty?

I have some 'vintage' mp3 players that only work with ID3v1 tags. 

Before I copy the V2 to V1 on tens of thousands of mp3 files it occurs to me I might have some that only have ID3v1 information, in which case copying the (blank) ID3v2 may wipe the tag.

I have run a test by creating an MP3 with only v1 tags and no v2 and it looks like it doesn't overwrite the v1 with blanks but I thought I would check...!

Sorry for the dumb question.
Many Thanks
Andy

**Comment by nicfit** [comment, marked as answer]
@megalithicportal , writing v1 does not touch v2, and vice-versa. You can have both as well.
